ios/RCTVideo.m

```diff
@@ -117,6 +117,15 @@ - (RCTVideoPlayerViewController*)createPlayerViewController:(AVPlayer*)player
     viewController.view.frame = self.bounds;
     viewController.player = player;
     viewController.view.frame = self.bounds;
+    
+    UIView *contentView = [viewController.view valueForKey:@"contentView"];
+    contentView.gestureRecognizers = [contentView.gestureRecognizers filteredArrayUsingPredicate:[NSPredicate predicateWithBlock:^BOOL(id  _Nullable evaluatedObject, NSDictionary<NSString *,id> * _Nullable bindings) {
+        if ([evaluatedObject isKindOfClass:UITapGestureRecognizer.class]) {
```

Comment:
@medlJeremy Instead of just allowing for gestures that are instances of the `UITapGestureRecognizer` class, it might be better if we look for `UIPinchGestureRecognizer` instances and disabling those like so:

    if ([evaluatedObject isKindOfClass:UITapGestureRecognizer.class]) {
        return NO;
    }
    return YES;

This way, we will only be blocking the pinch gesture recognizers which is what [this ticket](https://medlenterprises.atlassian.net/browse/NE-359) describes. Because iOS's `AVPlayer` or this `RCTVideo` class might be adding other gestures that may be internally required by the player. Thoughts?

Reply:
probably a good idea. Ill make the change